deftree/svcparse/parser.go

```diff
@@ -360,29 +504,54 @@ func ParseBindingFields(lex *SvcLexer) ([]*Field, error) {
 			lex.UnGetToken()
 			break
 		}
+		// Use recursion to parse custom HTTP verb sections
+		if val == "custom" {
+			err := fastForwardTill(lex, "{")
+			if err != nil {
+				return nil, nil, err
+			}
+			c, _, err := ParseBindingFields(lex)
+			if err != nil {
+				return nil, nil, err
```

Comment:
Would wrapping these errors be helpful? 
```
errors.Wrap(err, "error parsing custom binding field")
```

Reply:
I don't know if it would be materially helpful, but it certainly makes things more consistent.